sys/modules/efirt/Makefile

```diff
@@ -10,6 +10,9 @@ SRCS+=  device_if.h bus_if.h clock_if.h
 
 .if ${MACHINE_CPUARCH} == "amd64"
 SRCS+=	opt_hwpmc_hooks.h opt_kstack_pages.h
+.endif
+
+.if ${MACHINE_CPUARCH} == "aarch64" || ${MACHINE_CPUARCH} == "amd64"
```

Comment:
Hmm, I guess you'd have to use `MACHINE_ARCH == "aarch64c"` instead here (if MACHINE_ARCH is correct for purecap kernels).